internal/pkg/service/buffer/api/service/mapper/conditions.go

```diff
@@ -0,0 +1,34 @@
+package mapper
```

Comment:
Api `mapper` som:
- rozdelil do viacerych suborov
- odstranil pretypovanie, kde uz nie je potrebne
- fixol pointers, niekdy chybali, niekde boli zbytocne a komplikovalo to kod v service
- interne metody som spravil private